src/actions/authRole/authRole.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * AuthRole actions.
+ * @module actions/authRole/authRole
+ */
+
+import { AUTH_ROLE } from '@plone/volto/constants/ActionTypes';
+
+export function authenticatedRole(role) {
```

Comment:
Why do we need it? And what does it do? Why is it persistent?

Reply:
I created a new action-reducer pair named `AUTH_ROLE` for keeping the authenticated role available (persistant) throughout the userSession. As the other roles are dependent on this, it needs to be available on initial mount of the controlpanel.
<img width="448" alt="Screenshot 2021-04-08 at 2 13 36 PM" src="https://user-images.githubusercontent.com/22280901/113996245-9fd38580-9874-11eb-821d-2720a99c646b.png">

At the first place I haven't used it as "persistant", causing abnormal behaviour when updating authenticated roles. As this also responsible for behaving as "inherited" roles for users.